development/components/smarty-extensions/_index.md

```diff
@@ -96,9 +96,9 @@ So far, it is only used for forms (customer information and checkout).
 ### {url}
 
 This helper is used to generate URLs.
-This will take care of SSL, domain name, virtual and physical base URI, parameters concatenation, and of course URL rewriting.
+This will take care of http scheme (`http` or `https`), domain name, virtual and physical base URI, parameters concatenation, and of course URL rewriting.
```

Comment:
```suggestion
This will take care of the HTTP scheme (`http` or `https`), domain name, virtual and physical base URI, parameters concatenation, and URL rewriting.
```